### ISSUE TITLE
Log format not recognized after file cleanup

### ISSUE THREAD

**Issue by mfilippov** [JetBrains/ideolog#108]
When I cleanup log file I see "Log format not recognized" when new lines appear issue still there. Reopen file doesn't help. To fix I need enable disable pattern.

**Comment by mfilippov** [comment]
repro https://recordit.co/mvYb7fNV0H

**Comment by eugenemorozov-jb** [comment]
Sample log:
```
[08-Mar-2018 09:05:03 UTC] PHP Notice:  Trying to get property of non-object in /home/mysite/public_html/wp-includes/class-wp-query.php on line 3736
[08-Mar-2018 09:05:03 UTC] PHP Notice:  Trying to get property of non-object in /home/mysite/public_html/wp-includes/class-wp-query.php on line 3738
[08-Mar-2018 09:05:03 UTC] PHP Notice:  Trying to get property of non-object in /home/mysite/public_html/wp-includes/class-wp-query.php on line 3740
[08-Mar-2018 09:05:03 UTC] PHP Notice:  Trying to get property of non-object in /home/mysite/public_html/wp-includes/class-wp-query.php on line 3736
[08-Mar-2018 09:05:03 UTC] PHP Notice:  Trying to get property of non-object in /home/mysite/public_html/wp-includes/class-wp-query.php on line 3738
[08-Mar-2018 09:05:03 UTC] PHP Notice:  Trying to get property of non-object in /home/mysite/public_html/wp-includes/class-wp-query.php on line 3740
[08-Mar-2018 09:05:03 UTC] PHP Warning:  Cannot modify header information - headers already sent by (output started at /home/mysite/public_html/wp-includes/class-wp-query.php:3736) in /home/mysite/public_html/wp-includes/pluggable.php on line 1216
```
Message pattern: `^\[([^\]]+)\]\s+(PHP.*?)[.:](.*?)?(\sin\s)?(/.*?)?(\son\s)?(line.*)?$`
Message pattern start: `^\[`
Date format: `dd-MMM-yyy HH:mm:ss`
Time capture group: 1
Category capture group: 2

**Comment by ArtemKar123** [comment]
Fixed in #175. Will be released in the next version. Thanks for your feedback!
